src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Color {
```

Comment:
Перед тем как наследоваться от какого-то класса, нужно проверить, можем ли мы сказать, что один класс "является" другим классом (is a). Например, "круг" является "фигурой". Но "круг" не является "цветом". Можно сказать, что "круг" содержит "цвет", но не является "цветом".
Здесь классу Circle нужно наследоваться от класса Figure
```suggestion
public class Circle extends Figure {
```